src/IPM/callbacks.jl

```diff
@@ -1,9 +1,8 @@
-
 function eval_f_wrapper(ips::InteriorPointSolver, x::Vector{Float64})
     nlp = ips.nlp
     cnt = ips.cnt
     @trace(ips.logger,"Evaluating objective.")
-    cnt.eval_function_time += @elapsed obj_val = (get_minimize(nlp) ? 1. : -1.) * obj(nlp,view(x,1:get_nvar(nlp)))
+    cnt.eval_function_time += @elapsed obj_val = (get_minimize(nlp) ? 1. : -1.) * obj(nlp,unsafe_wrap(Vector{Float64},pointer(x),get_nvar(nlp)))
```

Comment:
To improve readability, I would move the `unsafe_wrap` in a line before the function call. Something like
```julia
x_primal = unsafe_wrap(Vector{Float64},pointer(x),get_nvar(nlp))
```